Templates/Clean Swift/Scene.xctemplate/UITableViewControllerXIB/___FILEBASENAME___Controller.swift

```diff
@@ -45,6 +45,7 @@ private extension ___VARIABLE_sceneName___Controller {
     private func setupTableView() {
         //TODO: Register custom cell
 //        tableView.register(cellType: ___VARIABLE_sceneName___TableViewCell.self)
+        tableView.reloadData()
```

Comment:
```suggestion
```

you don't need this, as setupTableView is called in viewDidLoad

Reply:
Sorry that was mistake.

Reply:
I was going to put that line into display... method :/